### DISCUSSION TITLE
Unable to insert dropdown list more than 256 char length in angular excel export using exceljs

### DISCUSSION THREAD

**Discussion by SriniR01** [exceljs/exceljs#1741] [Q&A]
Unable to have list of values more than 256 char length in excel export using exceljs and throw error and doesn't load any data to the dropdown column in excel.

Maximum list allowed or loaded is below 256 characters.
<img width="960" alt="Issue_256_charc_length" src="https://user-images.githubusercontent.com/80360949/120857606-7c187d00-c54f-11eb-992b-add81a6b23a8.PNG">

Could you please help me how to load few hunderd list of entries as dropdown.

**Comment by Saurabh-tcs** [comment]
I am also stuck at the same problem
